### PR TITLE
Fix "Center the full picture" on user avatar upload

### LINKED ISSUE #1130: Could not convert string to float
Hello
we get the error could not convert string to float when upload image and try to center it.

https://www.data.gouv.fr/fr/

this error also appears on our server

see attached file

could_not_convert_string_to_float

### PR DESCRIPTION
**Pull request by abulte**
Fix #1130

### CODE REVIEW

File: js/components/widgets/image-picker.vue [hunk 1]
```diff
@@ -133,7 +133,10 @@ export default {
             if (this.HAS_FILE_API) {
                 const data = {};
                 try {
-                    data.bbox = this.$refs.thumbnailer.bbox;
+                    let bbox = this.$refs.thumbnailer.bbox;
```
Comment (noirbizarre) [inline js/components/widgets/image-picker.vue]: Shoumd be a `const` instead of a `let`

Comment (abulte) [reply js/components/widgets/image-picker.vue]: Yes :-/ Worth a new PR?

Comment (noirbizarre) [reply js/components/widgets/image-picker.vue]: Nop, this shouldn't have been merged: breaks on lower resolutions (non-retina) and mobile :/

Comment (abulte) [reply js/components/widgets/image-picker.vue]: Really? :-( How? I just tested on mobile (retina) and it looks ok.

Comment (noirbizarre) [reply js/components/widgets/image-picker.vue]: No sorry, wrong window ;)
The PR is welcomed for the `s/let/const/`

Comment (abulte) [reply js/components/widgets/image-picker.vue]: 😌

**Pull request merged**